### ISSUE TITLE
Making the site RESPONSIVE

### ISSUE THREAD

**Issue by SuhasBRao** [mamamia5x/markdownpedia#40]
Hey there, I was just checking the site and the thing I noticed is that the site is not yet ***Responsive***. I don't know whether we can make the site responsive using markdown or not. 
I did not find any issues relating to this, if there is any open issue relating to the same do let me know.

Here's the mobile view of the site,(Uploading the screenshot of the site, please bear if the image is a bit enlarged)
![mobile view](https://user-images.githubusercontent.com/65769327/126072178-fbbbd645-4139-41a5-a721-aec9c39d283f.jpeg)

We can work on it. Please do comment your views.

**Comment by mamamia5x** [comment]
Yeah, mobile HTML is not one of my strong suits. I'm trying to write a script for the site's back-end, which in theory would make managing the site so much easier. I don't really have much free time, and I can't find the right way to actually program my ideas in. It'll be nice to see a more responsive website, and fixing this would not require the back-end (if I make changes to the back-end, the changes to fix this issue would not be affected)

Also, Markdown probably wouldn't fix this issue. I think the CSS would have to change. It could be possible to change some of the issue markdown wise, but it wouldn't be good for a new user making their own page, not knowing the requirements needed to make it look right on a mobile device.

**Comment by SuhasBRao** [comment]
Thanks for replying, I would like to work on this. Please assign the issue to me.

**Comment by mamamia5x** [comment]
Sorry for the long response.

**Comment by SuhasBRao** [comment]
Thanks for assigning, It's ok i'll start working on this issue.

**Comment by yamanidev** [comment]
I'd like to work on this if possible, I will not use any 3rd party library. Just pure Sass (or vanilla CSS if you wish to).

**Comment by RamGoel** [comment]
Hi, as this is my first contribution so i think that this issue is good to start, kindly allow me to work on this issue if possible.

**Comment by mamamia5x** [comment]
Sure thing, you can work on this issue.

**Comment by RamGoel** [comment]
Here is the Mobile Preview of the Site.
@SuhasBRao Have a Look at this Preview.
@mamamia5x  Kindly assign me the issue so that i can submit the code.

![Screenshot 2021-11-10 at 15-03-16 Markdownpedia - Home](https://user-images.githubusercontent.com/71088429/141148010-0f1d5b02-c0bf-4dca-b01a-9786f2e95b9b.png)

**Comment by ramantamang777** [comment]
We can even make a toggle bar to the top right!
 Can you assign this to me to work on it?

**Comment by MeetaHaldar** [comment]
can i do some contribution???

**Comment by YasarArafathAli** [comment]
Hey!!
are you willing to accept new contributors? I am interested in contributing to this repository as I am proficient with vanilla CSS, it would be great.
Thanks

**Comment by RamGoel** [comment]
> Here is the Mobile Preview of the Site. @SuhasBRao Have a Look at this Preview. @mamamia5x Kindly assign me the issue so that i can submit the code.
> 
> ![Screenshot 2021-11-10 at 15-03-16 Markdownpedia - Home](https://user-images.githubusercontent.com/71088429/141148010-0f1d5b02-c0bf-4dca-b01a-9786f2e95b9b.png)

Anything can be done?

**Comment by SuhasBRao** [comment]
It looks good

**Comment by Prashant7451** [comment]
Hey, @SuhasBRao  i want to fix this problem

**Comment by AjaySureshPatil** [comment]
I would like to work on this project please assign me

**Comment by jalonghurst** [comment]
Hi, I would be happy to contribute on this issue if possible!

**Comment by dotarjun** [comment]
Is this issue being worked on? I'd like to contribute wherever I can
